npm/css-src/main.css

```diff
@@ -0,0 +1,4337 @@
+.govuk-link {
```

Comment:
Not sure we should be merging the main.css file as it is generated from the sass files.

Reply:
Good point, I've deleted it.

Reply:
I've removed main.js while I was at it as that's the same problem.